devicetypes/smartthings/zwave-fan-controller.src/zwave-fan-controller.groovy

```diff
@@ -111,7 +111,7 @@ def fanEvents(physicalgraph.zwave.Command cmd) {
 	def result = [createEvent(name: "switch", value: value)]
 	result << createEvent(name: "level", value: cmd.value == 99 ? 100 : cmd.value)
 	def fan_level = Math.round(cmd.value/33)
-	if (cmd.value < 33 * cmd.value > 1) fan_level = 1 //sometimes we get "1" when the device is on low
+	if (cmd.value < 33 && cmd.value > 1) fan_level = 1 //sometimes we get "1" when the device is on low
```

Comment:
You'll either want `cmd.value > 0` or `cmd.value >= 1` if we get sent `1` from the device.

Reply:
Very true!